cordova-js-src/exec.js

```diff
@@ -60,18 +60,32 @@ module.exports = function (success, fail, service, action, args) {
             // CB-5806 [Windows8] Add keepCallback support to proxy
             onSuccess = function (result, callbackOptions) {
                 callbackOptions = callbackOptions || {};
+                var callbackStatus;
+                if (callbackOptions.status !== null && callbackOptions.status !== undefined) {
```

Comment:
So resolution sounded like if (callbackOptions.hasOwnProperty('status')) ?  That ensures that even for falsy values, if they were set, they're propagated through as-is?